From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Thomas Clark <tjclark@wpi.edu>
Date: Fri, 25 Jul 2014 09:41:06 -0400
Subject: [PATCH 0200/6262] AnalogInput::PIDGet returns an average
 voltage[artf2391]

Previously it returned a raw value instead of a voltage.

Change-Id: Ia7f06ca24a6c79468c7be89a07c615d7015ffef9
---
 wpilibc/wpilibC++/lib/AnalogInput.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wpilibc/wpilibC++/lib/AnalogInput.cpp b/wpilibc/wpilibC++/lib/AnalogInput.cpp
index a331cbb3d128c7a9c95faa311dadb7ec500a63c3..fe4d3807d5f008bca5d9d671b932fd6eeed5c307 100644
--- a/wpilibc/wpilibC++/lib/AnalogInput.cpp
+++ b/wpilibc/wpilibC++/lib/AnalogInput.cpp
@@ -398,7 +398,7 @@ float AnalogInput::GetSampleRate()
 double AnalogInput::PIDGet()
 {
 	if (StatusIsFatal()) return 0.0;
-	return GetAverageValue();
+	return GetAverageVoltage();
 }
 
 void AnalogInput::UpdateTable() {
